.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This folder should also be in the .gitignore

Reply:
I have successfully added .vscode file directory inside .gitignore